sdk/spring/scripts/replace_artifact.py

```diff
@@ -18,6 +18,20 @@
         'pom_list': [
```

Comment:
Rename `replace_artifact.py` to `replact_util.py`, because now it also update version, not only update artifact.